daemon/web/pages/containers/Containers.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import TerminalView from '../../components/TerminalView/TerminalView';
+
 
 const styles = {
```

Comment:
can you remove this since we're not using it?

Reply:
actually there looks like a reference to a style that's supposed to be here, line 27:

```js
return (
    <div style={styles.container}>
```

Probably remove both this object and the reference